TeamShiksha/logoexecutive
# feat/destoyapikeys

### Summary

Adding API endpoint for deleting API keys

### Description

This PR adds functionality to the API that allows deletion of API keys based on their unique identifier.

### How to Test the Changes

* Use a tool like Postman or cURL to send a DELETE request to the endpoint.
* Use the URL: http://localhost:3000/keys/delete?keyId=keyid, replacing keyid with the actual key identifier.
* Verify the response code is 200 for successful deletion.
* Ensure the deleted key is no longer accessible when queried.


### Context (Optional)

This addition is crucial for better management of API keys within the system. It enhances security by providing a way to revoke access to compromised or outdated keys.

### Screenshots or Recordings (Optional)

![image](https://github.com/FrontendArmy/logoexecutive-backend/assets/99191853/157201f0-b4ae-4186-bfba-7858545d73db)

![image](https://github.com/FrontendArmy/logoexecutive-backend/assets/99191853/6ae31cc8-2b59-406d-a8a4-28849b31c8d2)

![image](https://github.com/FrontendArmy/logoexecutive-backend/assets/99191853/70afcc75-22b8-41a7-bdc5-cba4d02523c6)

## Checklist
